cmake/find/Findodb.cmake

```diff
@@ -0,0 +1,108 @@
+# ----------------------------------------------------------------------
```

Comment:
Yes, I believe you are right, it is hardcoding the path somewhere, because it is failing at "./configure"  stage of odb-sqlite, with an error like:
dyld: Library not loaded: /home/myhome/.../.hunter/.../Build/.../libodb.dylib
  Referenced from: /home/myhome/../.hunter/.../Build/.../conftest
  Reason: image not found
Trace/BPT trap:5

(sorry I don't have the exact message with me right now)

Maybe it was something to do with RPATH, I have to read it up to understand it better.

I will separate the example at make a pull request at the end then, since the example relies in the compiler and `odb` package by itself does nothing, and was there to test `hunter_autotools_*`.